### PR TITLE
0 is not positive or negative

### LINKED ISSUE #322: 0 is not a positive number
`.positive()` accepts 0, but shouldn't 0 not be considered a positive number?

### PR DESCRIPTION
**Pull request by captDaylight**
fixes #322

### CODE REVIEW

**Comment by jquense** [comment]
thanks!

File: test/number.js [hunk 1]
```diff
@@ -262,8 +266,10 @@ describe('Number types', function() {
 
       v
         .isValid(0)
-        .should.eventually()
-        .equal(true),
+        .should.be.rejected()
+        .then(null, function(err) {
+          err.errors[0].should.contain('this must be a positive number');
```
Comment (jquense) [inline test/number.js]: this isn't hit i don't think. `should.be.rejected` will catch the error and return it instead of letting it propagate a believe

Comment (captDaylight) [reply test/number.js]: yup my bad, I did it without testing locally. Commit coming down the pipeline...

**Comment by captDaylight** [comment]
Tests pass, but I see that "should check integer" test seems to be a duplicate of the positive tests. Should I fix that now?

**Comment by captDaylight** [comment]
good to go!

**Comment by jquense** [comment]
thanks!

**Comment by captDaylight** [comment]
awesome!

**Pull request merged**